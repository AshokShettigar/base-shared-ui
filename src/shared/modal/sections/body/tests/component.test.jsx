import React from 'react';
import { shallow } from 'enzyme';

import ModalBodySection from '../component';

const defaultProps = {
    sidebar: (<span>sidebar content</span>),
    children: (<span>children content</span>),
    node: (<span>node content</span>)
};

it('should render component', () => {
    const wrapper = shallow(<ModalBodySection {...defaultProps} />);

    expect(wrapper).toMatchSnapshot();
});
