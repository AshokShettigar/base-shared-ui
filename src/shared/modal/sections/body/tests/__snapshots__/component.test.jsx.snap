// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
<div
  className="root"
  data-component="ModalBodySection"
>
  <div
    className="content"
  >
    <span>
      children content
    </span>
  </div>
  <ModalSideBarSection>
    <span>
      sidebar content
    </span>
  </ModalSideBarSection>
</div>
`;
